angelsrefining/locale/en/ore-refining-refining.cfg

```diff
@@ -97,10 +97,10 @@ loc-angels-copper-slag=Copper slag
 angels-ore8-sludge=Ferrous sludge
 angels-ore9-sludge=Cupric sludge
 ; tier 3.5 refining
-angels-ore8-solution=Ferrous slurry
+angels-ore8-solution=Ferrous solution
```

Comment:
im not 100% by this change, i like the slurry term as it implies a thick solid laiden fluid, which would be more correct in this case

Reply:
@Pezzawinkle  Can you read and comment on my ideas in #370 regarding "slurry" and "solution"?

Reply:
I do agree that sludge shoud be before slurry in most cases, I think the problem here, is that the solids are repeatedly wet then dried... i will take a new look processes again in more detail in the next few weeks.